docs/cli.html

```diff
@@ -790,18 +790,27 @@ <h2 id="migrate">migrate</h2>
+<p>Migrate a percentage of users or a single user from one version of your integration to another.</p>
+</blockquote><p><strong>Usage</strong>: <code>zapier migrate FROMVERSION TOVERSION [PERCENT]</code></p><p>Start a migration to move users between different versions of your integration. You may also &quot;revert&quot; by simply swapping the from/to verion strings in the command line arguments (i.e. <code>zapier migrate 1.0.1 1.0.0</code>).</p><p><strong>Only use this command to migrate users between non-breaking versions, use <code>zapier deprecate</code> if you have breaking changes!</strong></p><p>Migration time varies based on the number of affected Zaps. Be patient and check <code>zapier jobs</code> to track the status. Or use <code>zapier history</code> if you want to see older jobs.</p><p>Since a migration is only for non-breaking changes, users are not emailed about the update/migration. It will be a transparent process for them.</p><p>We recommend migrating a small subset of users first, via the percent argument, then watching error logs of the new version for any sort of odd behavior. When you feel confident there are no bugs, go ahead and migrate everyone. If you see unexpected errors, you can revert.</p><p>You can migrate a specific user&apos;s Zaps by using <code>--user</code> (i.e. <code>zapier migrate 1.0.0 1.0.1 --user=user@example.com</code>). This will migrate Zaps in any account the user is part of where the following criteria is met.  </p><ul>
+<li><p>The Zap is owned by the user.  </p>
+</li>
+<li><p>The Zap is not shared.  </p>
+</li>
+<li><p>The integration auth used is not shared.  </p>
+</li>
+</ul><p>Alternatively, you can pass the <code>--account</code> flag, (i.e. <code>zapier migrate 1.0.0 1.0.1 --account=account@example.com</code>). This will migrate all the user&apos;s Zap steps in any account the user is part of, regardless of Zap owner.</p><p><strong>The <code>--account</code> flag should be used cautiously as it can break shared Zaps for other users in Team or Company accounts.</strong>  </p><p>You cannot pass both <code>PERCENT</code> and <code>--user</code> or <code>--account</code>.</p><p>You cannot pass both <code>--user</code> and <code>--account</code>.</p><p><strong>Arguments</strong></p><ul>
 <li>(required) <code>fromVersion</code> | The version FROM which to migrate users.</li>
 <li>(required) <code>toVersion</code> | The version TO which to migrate users.</li>
 <li><code>percent</code> | Percentage (between 1 and 100) of users to migrate.</li>
 </ul><p><strong>Flags</strong></p><ul>
-<li><code>--user</code> | Migrate only this user</li>
+<li><code>--user</code> | Migrate a users private Zaps using the integration</li>
+<li><code>--account</code> | Migrate all a users Zaps using the integration, incl. shared Zaps</li>
```

Comment:
Seems like this needs a bit more info about the effect of the `account` flag - maybe something like `Migrates all of the Zaps (for all users) within each of user's accounts`